backend/tests/Feature/UserAuthTest.php

```diff
@@ -47,4 +49,21 @@ public function testFailLogin()
                 'error'
         ]);
     }
+
+    public function testUpdateUser()
+    {
+        $user = factory(User::class)->create();
+        $token = JWTAuth::fromUser($user);
+        $headers = ['Authorization' => "Bearer $token"];
+
+        $response = $this->actingAs($user)->put('api/v1/users/me', [
+            'name' => 'Test',
+            'email' => 'test@unique.com',
+            'password' => ''
+        ], $headers);
+
+//        dd($response->json());
+
+        $this->assertEquals(200, $response->getStatusCode());
```

Comment:
also check database that user has changed. https://laravel.com/docs/5.8/database-testing#available-assertions